docs/docusaurus/src/css/typography.scss

```diff
@@ -0,0 +1,80 @@
+:root {
+    --ifm-font-family-base: 'Source Sans Pro', 'Open Sans', Montserrat, Verdana, sans‑serif;
+
+    --h1-font-size: 3rem;
+    --h2-font-size: 2rem;
+    --h3-font-size: 1.75rem;
+
+    --ifm-h1-font-size: var(--h1-font-size);
+    --ifm-h2-font-size: var(--h2-font-size);
+    --ifm-h3-font-size: var(--h3-font-size);
+
+    --h4-font-size: 1.5rem;
+    --h5-font-size: 1.25rem;
+
+    --p-font-size: 1.125rem;
+    --p-lg-font-size: 1.5rem;
+    --p-md-font-size: 1rem;
+    --p-sm-font-size: 0.875rem;
+    --p-tiny-font-size: 0.75rem;
+
+    --ifm-heading-line-height: 1.25;
+    --p-line-height: 1.5;
+
+    --ifm-font-weight-semibold: 600;
+
+    --ifm-heading-margin-bottom: 0.3em;
+    --ifm-heading-vertical-rhythm-bottom: 0;
+
+    --ifm-list-item-margin: 0.5em;
+
+    --ifm-dropdown-font-weight: bold;
+}
+
+/* override huge h1 */
+.h1Heading_node_modules-\@docusaurus-theme-classic-lib-next-theme-Heading-styles-module {
+    font-size: var(--ifm-h1-font-size) !important;
+}
+
+.markdown > h1 {
+    --ifm-h1-font-size: var(--h1-font-size);
```

Comment:
This rule might feel a little redundant but the original CSS built by infima overrides the `--ifm-h1-font-size` variable with this particular rule so the new size is not applied to the h1s in some of the cases by just overriding the variable in :root. This also happens with h2 and h3, so that's the reason why these three rules exist